src/controller/user-controller.ts

```diff
@@ -133,6 +133,14 @@ export default class UserController extends BaseController {
           restrictions: { acceptedTOS: false },
         },
       },
+      '/nfc/:id': {
```

Comment:
Calling this parameter `id` was pretty confusing to me at first. Maybe rename it to something else like `nfcCode` or something, so it is also not confusing in generated clients.